libs/ui/src/ListBox/ListBox.module.css

```diff
@@ -0,0 +1,35 @@
+.list {
+  color: var(--ams-color-primary-black);
+  font-family: var(--ams-text-font-family);
+  font-size: var(--ams-text-level-5-font-size);
+  font-weight: var(--ams-text-font-weight-normal);
+  line-height: var(--ams-text-level-5-line-height);
+  list-style-type: none;
+  margin-block: 0 var(--ams-space-grid-md);
+  padding-inline: 0;
+}
+
+.option {
+  box-shadow:
+    /* Left border */
+    inset var(--ams-border-width-sm) 0 0 0 var(--ams-color-primary-black),
+    /* Right border */ inset calc(var(--ams-border-width-sm) * -1) 0 0 0 var(--ams-color-primary-black),
+    /* Bottom border */ inset 0 calc(var(--ams-border-width-sm) * -1) 0 0 var(--ams-color-primary-black);
```

Comment:
These two comments are inline, better to put them above like the Left Border.

Reply:
Prettier formats them like this when you place the comments above the CSS rule. I've inlined them after the rule now, which is more readable imo. Ok with you?